lib/pavilion/commands/result.py

```diff
@@ -6,6 +6,7 @@
 import pathlib
 import pprint
 import shutil
+from tkinter import Y
```

Comment:
This isn't used (and even if it was, yikes).

Reply:
Fixed. Yikes indeed.  I definitely did that and did not do it intentionally.  Must have been a vscode autocomplete issue.  I'll have to watch for that in the future.